src/app/navbar/navbar.component.scss

```diff
@@ -0,0 +1,89 @@
+@import '../../styles/partials/variables';
+
+.navbar {
+	background-color: $background-color-navbar;
```

Comment:
I don't think that there should be variable for navbar, as it is going to be used only here not anyplace else, so I guess you can add only the `rgb()` values, as this is not going to be used at anyplace else.